smallrye-reactive-messaging-amqp/pom.xml

```diff
@@ -56,6 +56,12 @@
       <version>${artemis.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.14.2</version>
```

Comment:
Should we add this to the dependency management? We have other modules using jackson no?